src/scenes/Jarjestajat/Jarjestaja/Hakemukset/Muutospyynto/components/MuutospyyntoWizard.js

```diff
@@ -433,6 +427,12 @@ const MuutospyyntoWizard = ({
     setPage(parseInt(match.params.page, 10));
   }, [match.params.page]);
 
+  useEffect(() => {
+    if(muutoshakemus.readyToCloseWizard === true) {
+      setTimeout(closeWizard,2000);
```

Comment:
Why is this timeout needed?

Reply:
It's a crutch. Allows for the "Muutospyyntö tallennettu!" Toast to run before closing the wizard.